com.github.ImCompressor.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "com.github.ImCompressor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "imcompressor",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
```

Comment:
what does this need the network for?

Reply:
Indeed, not necessary. Fixed in https://github.com/Huluti/ImCompressor/commit/029528948d034a8dc28605c35e5ed3f1b3af581c.